### PR TITLE
simple algorithms

### PR DESCRIPTION
**Pull request by MadhuBeesetty**

### CODE REVIEW

File: simple-algorithms/arraysum.js [hunk 1]
```diff
@@ -21,3 +21,16 @@ console.log(b);*/
...
+const max=(arr,i=0,maxx=arr[0])=>{
+  if(arr[i]>maxx){
+    maxx=arr[i]
+  }
+  if(i===(arr.length-1)){
+    return maxx;
+  }
+  return max(arr,i+1,maxx);
+}
+console.log(max([5,26,15]));
```
Comment (songz) [inline simple-algorithms/arraysum.js:34]: does this test case work? `[5,3,8]`

File: simple-algorithms/string.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+//printing a string n number of times//
+const print = (a,b,i=1,c=" ")=>{
```
Comment (songz) [inline simple-algorithms/string.js:2]: wrong function. You need to return for this function (not print)

File: simple-algorithms/sum.js [hunk 3]
```diff
@@ -6,11 +6,21 @@ a.forEach((e)=>{
...
   });
   return(b);
 };
 console.log(sum([1,2,3,4,5],0));
+*/
+
+//using recursion
 
+const add =(arr,sum=0,i=0)=>{
+if(arr.length == i){
```
Comment (songz) [inline simple-algorithms/sum.js]: please indent.

File: simple-algorithms/arraysum.js (hunk 1 above)
Comment (MadhuBeesetty) [reply simple-algorithms/arraysum.js:34]: yes it is working

**Pull request merged**